### ISSUE TITLE
Highlighting of the `todo.txt` file

### ISSUE THREAD

**Issue by dk949** [arnarg/todotxt.nvim#13]
Hi 👋 

Does the plugin provide highlighting for the `todo.txt` file itself (as shown in the asciinema gif), or is it only for the side bar? I didn't see any ftdetect code in the repo, but then again I'm not too familiar with lua plugins.

Cheers.

**Comment by arnarg** [comment]
Around the time I made this plugin I also made a [tree-sitter parser for todo.txt](https://github.com/arnarg/tree-sitter-todotxt). I also already added it to [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter/pull/2594).
I have not been using this plugin nor the tree-sitter parser for a while now but as far as I remember you just need a ftdetect that sets the filetype to `todotxt` for it to work (or `set filetype=todotxt`).

I hope this helps.
